{"issue_type": "request_error", "error_msg": "HTTPSConnectionPool(host='selma-al.gov', port=443): Max retries exceeded with url: /robots.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1b31d804c0>, 'Connection to selma-al.gov timed out. (connect timeout=15)'))"}